src/lib/features/export-import-toggles/export-import.e2e.test.ts

```diff
@@ -128,26 +128,15 @@ const createSegment = (postData: UpsertSegmentSchema): Promise<ISegment> => {
 };
 
 const createContextField = async (contextField: IContextFieldDto) => {
-    await app.request.post(`/api/admin/context`).send(contextField).expect(201);
+    await app.api.createContextField(contextField).expect(201);
```

Comment:
this is breaking law of demeter as we're peeking inside the app.api. I'd prefer to call function: createContextApi(app, contextField, 201) or createContextApi(app)(contextField, 201)

Reply:
Yeah, this is a controversy I had as well, but it's not necessarily a break inside the app, but more like a chain of functions.

Personally, I have my doubts on how to evolve what you propose when we have to deal with errors, or if you want to check headers, or maybe cookies...

Reply:
I still prefer `app.createContext(contextField)` because the app is already imported so it serves as an entry point. I've hidden the expected response code using default values for the arguments (that way you don't need to know the specifics of each API, because we're not that consistent with response codes), but still keep the composability